### ISSUE TITLE
I have a question about LidarTestingUtility.

### ISSUE THREAD

**Issue by tmdwhsh** [iliasam/OpenTOFLidar#19]
Hi, iliasam.
I have uploaded 'not_scanning_fw_pcb_v4.hex' and the
and noticed that the LED is blinking.
I also checked the 'Laser_Pulse' of the TDC and found that a pulse of about 3V is generated.
I also checked that the voltage of the Vlaser is changing the same as what I set in the Testing Utility.
However, if you look at the picture I attached, you can see that many of the items in the Rangefinder State have been changed to
indicating a 'N/A' state.
Since the voltage is set normally and various values are recorded through 'Start', it seems that the serial communication is working well, but can you tell me why 'N/A' appears?
![KakaoTalk_20230718_213547890](https://github.com/iliasam/OpenTOFLidar/assets/43981588/55456d87-5d1c-45ed-85f2-5a186a711de8)

**Comment by iliasam** [comment]
Hello. Can you tell me what is the release of firmware that you are using?

**Comment by tmdwhsh** [comment]
I made a board with the PCB v4 design and uploaded 'release2 - not_scanning_fw_pcb_v4.hex'.
----------------------------------------------------------------------------------------------------

![image](https://github.com/iliasam/OpenTOFLidar/assets/43981588/a80f6631-ecf5-40e8-857e-5922e18cd528)
After comment, I uploaded 'release1 - not_scanning_fw_pcb_v4.hex' and Same problem,,,
----------------------------------------------------------------------------------------------------

![image](https://github.com/iliasam/OpenTOFLidar/assets/43981588/6e47c3fe-6ad6-4594-8c8e-9c3588ce9f3d)
I re-uploaded 'release1 -i not_scanning_fw_pcb_v4.hex' and the N/A issue went away, but I got a TDC error with no 'laser_pulse'.

**Comment by iliasam** [comment]
Testinng Utility is decigned for beeing used with firmware from "Release 2" so it is strange that you are getting state data from "Release 1".
You can try to use this dll with "Testinng Utility".
[mavlink.zip](https://github.com/iliasam/OpenTOFLidar/files/12092707/mavlink.zip)

**Comment by tmdwhsh** [comment]
I figured out my problem.
When I uploaded Release1, the laser works and N/A appears, but the
When I uploaded Release2, the laser doesn't work, but the N/A is gone.

Any idea what I can do to fix the above (for the laser to work in Release2 as it did in Release1)?

I'll try to test with your DLL file and Release1 first. 
Thanks a lot!!!

+ I replaced the mavlink.dll in TestingUtility with the file you attached and uploaded release1, but I still have the issue with it showing N/A. 
+ "I re-uploaded 'release1 -i not_scanning_fw_pcb_v4.hex' and the N/A issue went away, but I got a TDC error with no 'laser_pulse'."-> This was my mistake. Release2 is correct.
